fe/fe-core/src/main/java/org/apache/doris/tablefunction/ExternalFileTableValuedFunction.java

```diff
@@ -128,6 +124,21 @@ public abstract class ExternalFileTableValuedFunction extends TableValuedFunctio
     private boolean trimDoubleQuotes;
     private int skipLines;
     private long tableId;
+    private static final ImmutableSet<TFileFormatType> HIVE_FILE_FORMATS = new ImmutableSet.Builder<TFileFormatType>()
+            .add(TFileFormatType.FORMAT_RCBINARY)
+            .add(TFileFormatType.FORMAT_RCTEXT)
+            .add(TFileFormatType.FORMAT_SEQUENCE)
+            .build();
+
+    // When useMetastore is "false", users can specify column names and column types
+    // for file formats such as RCFile and SequenceFile that rely on metadata stored in Hive Metastore
+    private String useMetastore;
+    // Comma-separated list of column names.
+    // Only applicable when useMetastore is false and for formats like RCFile and SequenceFile.
+    private String columnNamesStr;
```

Comment:
Could you reuse `FileFormatUtils.parseCsvSchema()`?
Although it is called "csv schema", but actually it can be used for any file format which need specified schema